source/includes/gcp/_routers.md

```diff
@@ -0,0 +1,128 @@
+### Routers
+
+Cloud Router allows to programs custom dynamic routes and it scales with your network traffic. Cloud Router isn't a physical device that might cause a bottleneck. It can't be used by itself. But, it is required or recommended in the following cases:
+
+- Required for Cloud NAT
+- Required for Cloud Interconnect and HA VPN
```

Comment:
Would it make it more clear to use `high-availability (HA) VPN` instead of `HA VPN`?

Reply:
Fixed in https://github.com/cloudops/cloudmc-api-docs/pull/147/commits/05f7638a3c2e9aa60e658d66a47447a592ad7fae